java-frontend/src/main/java/org/sonar/java/se/checks/NullDereferenceCheck.java

```diff
@@ -222,7 +222,9 @@ public ProgramState checkPostStatement(CheckerContext context, Tree syntaxNode)
   }
 
   private static boolean isAnnotatedCheckForNull(MethodInvocationTree syntaxNode) {
-    return syntaxNode.symbol().metadata().isAnnotatedWith("javax.annotation.CheckForNull");
+    return syntaxNode.symbol().metadata().isAnnotatedWith("javax.annotation.CheckForNull")
```

Comment:
you can avoid calling two times `syntaxNode.symbol().metadata()` by storing it first.